news/2023/2023-05-13-new-featured-artist-cygnus.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "New Featured Artist: cygnus"
+date: 2023-05-13 12:00:00 +0000
+---
+
+Do you believe in simulation theory? If not, disregard this message and carry on with your day. Otherwise, you might want to check out our newest Featured Artist.
+
+![](https://assets.ppy.sh/artists/359/header.jpg)
+
+**cygnus** (known here on osu! as [Matrix](https://osu.ppy.sh/users/5052899)) is the embodiment of the circle clicker's dream: living in Japan and producing rhythm game songs all day every day.
+
+While he began as a pioneer of AR 11 gameplay and later shifted his attention towards mapping, he's recently discovered that music is his favourite form of expression. Inspired by creators like our very own [*-45*](https://osu.ppy.sh/beatmaps/artists/297), *onoken*, and *SHIKI*, **cygnus** pulls off a distinctly old-school style through both his compositions and nostalgic sample choices.
+
+**23** tracks are patiently waiting for your editor magic on [**cygnus**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/359)!
+
+Given **cygnus**'s presence in the mapping community, it's no surprise that he wrangled a fair share of the mapping community into creating maps for his music. Below are a few examples of the awesome maps that have been made in the last few months, including maps by the artist himself:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/359/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### cygnus - STRIKER+
+
+Play through [this soon-to-be-Ranked map](https://osu.ppy.sh/beatmapsets/1966073) with an 8-star top difficulty and guest difficulties by the *no caps crew*: [rrtyui](https://osu.ppy.sh/users/352328), [mithew](https://osu.ppy.sh/users/2571893), [semaphore](https://osu.ppy.sh/users/6313643), and [fanzhen0019](https://osu.ppy.sh/users/418699)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/The%20Evolution%20of%20War/cygnus%20-%20STRIKER%2B.mp3">
```

Comment:
this one is broken now here and on the website (seems to be still located at the old url https://assets.ppy.sh/artists/359/Evolution%20of%20War%2Fcygnus%20-%20STRIKER%2B.mp3)

Reply:
i think this is a caching thing on your end, will purge before merge just in case

Reply:
(checked, all fixed)